charts/kargo/templates/api/deployment.yaml

```
@@ -41,7 +41,7 @@ spec:
 {{- if .Values.api.adminAccount.enabled }}
             - name: ADMIN_ACCOUNT_ENABLED
               value: "true"
-            - name: ADMIN_ACCOUNT_PASSWORD
+            - name: HASHED_ADMIN_ACCOUNT_PASSWORD
```

```suggestion
            - name: ADMIN_ACCOUNT_PASSWORD_HASH
```

Consistent prefixes for related configs is preferable

---

Agreed

---

Fixed in cd2d7101c22a13e71453bd6bf9bf318f0de289d3